defaults/main.yml

```diff
@@ -31,9 +31,10 @@ nimbus_fluffy_user: 'nimbus'
 nimbus_fluffy_group: 'staff'
 
 # General
-nimbus_fluffy_network: 'mainnet' # or testnet0
+nimbus_fluffy_network: 'mainnet'
```

Comment:
I don't see the point of this variable now. Lets just use it as it is. We can change the default to `testnet0` if you want.